sdk/core/AzureCore/Source/ClientLogger.swift

```diff
@@ -44,7 +44,7 @@ public protocol ClientLogger {
     func warning(_: @autoclosure @escaping () -> String?)
     func error(_: @autoclosure @escaping () -> String?)
 
-    func log(_: @escaping () -> String?, atLevel: ClientLogLevel)
+    func log(_: () -> String?, atLevel: ClientLogLevel)
```

Comment:
It was unneeded. We don't store the closure so its scope never escapes the log method, thus @enclosing was unneeded. I think I had added it initially because it was in Colby's logger.